src/transformers/models/blip/modeling_blip.py

```diff
@@ -985,8 +985,9 @@ def forward(
 
         >>> url = "http://images.cocodataset.org/val2017/000000039769.jpg"
         >>> image = Image.open(requests.get(url, stream=True).raw)
+        >>> text = "A picture of"
 
-        >>> inputs = processor(images=image, return_tensors="pt")
+        >>> inputs = processor(images=image, text=text, return_tensors="pt")
```

Comment:
I am curious about `A picture of ...[what?]` Is it my favorite 🐱 😹 ?

Reply:
ahahah

Reply:
It's the model's responsibility to predict the correct word after "A picture of" :D

Reply:
Oh, I put at the wrong line. But I also realized this doc example is `forward` not `generate`, so no prediction.
(it might be a good idea to add `generate` example too, but I am fine with the current example)